content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -86,7 +86,7 @@ kubectl config --kubeconfig=config-demo set-credentials experimenter --username=
 ```
 
 {{< note >}}
-To delete a user you can run `kubectl config unset users.<name>`
+To delete a user you can run `kubectl --kubeconfig=config-demo config unset users.<name>`. To remove clusters and/or contexts run the same command, but change `users.<name>` to `clusters.<name>` and `contexts.<name>` respectively.
```

Comment:
Suggestion: Instead of saying  'remove clusters and/or contexts run the same command, but change `users.<name>` to `clusters.<name>` and `contexts.<name>` respectively', can you add line with respective commands? I

Reply:
@Kartik494, the first sentence on line 89 looks good.
As @savitharaghunathan suggested, it would be helpful to readers to see the full commands for the task.
I have not run the commands using the `config-demo` configuration file, but you could add two additional sentences such as:

To delete a user, you can run `kubectl --kubeconfig=config-demo config unset users.<name>`. 
To remove a cluster, you can run `kubectl --kubeconfig=config-demo config unset clusters.<name>`.
To remove a context, you can run `kubectl --kubeconfig=config-demo config unset contexts.<name>`.

Reply:
@savitharaghunathan @kbhawkey I have addressed the comments. Please check
Thanks